### ISSUE TITLE
NameError: name `print_ascii_tree` is not defined

### ISSUE THREAD

**Issue by maximousblk** [soxoj/maigret#103]
When using more than 30s of timeout, I get this error everytime

```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.2/x64/bin/maigret", line 33, in <module>
    sys.exit(load_entry_point('maigret==0.1.18', 'console_scripts', 'maigret')())
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/maigret.py", line 428, in run
    loop.run_until_complete(main())
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/maigret.py", line 348, in main
    results = await maigret(username=username,
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/checking.py", line 576, in maigret
    results = await executor.run(coroutines)
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/checking.py", line 55, in run
    results = await self._run(tasks)
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/checking.py", line 70, in _run
    return await asyncio.gather(*futures)
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/checking.py", line 203, in update_site_dict_from_response
    return sitename, process_site_result(response, query_notify, logger, results_info, site_obj)
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/checking.py", line 385, in process_site_result
    query_notify.update(result, site.similar_search)
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/notify.py", line 199, in update
    ids_data_text = get_dict_ascii_tree(self.result.ids_data.items(), ' ')
  File "/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/maigret/utils.py", line 70, in get_dict_ascii_tree
    field_value = print_ascii_tree(eval(field_value), prepend_symbols)
NameError: name 'print_ascii_tree' is not defined
```

Logs: [Generate Report.txt](https://github.com/soxoj/maigret/files/6247607/5_Generate.Report.txt)

**Comment by maximousblk** [comment]
Nevermind, it happens even with 30 second timeout

**Comment by soxoj** [comment]
Thank you. Fixed in #104, please check.

**Comment by maximousblk** [comment]
Yep, fixed!
